docker-compose.yml

```diff
@@ -49,6 +49,26 @@ services:
     networks:
       - doctrina_network
 
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.1.1
+    container_name: doctrina_elasticsearch
+    environment:
+      - ES_JAVA_OPTS=-Xmx512m -Xms512m -Dlog4j2.formatMsgNoLookups=true
+      - discovery.type=single-node
+    mem_limit: 6g
+    networks:
+      - doctrina_network
+
+  kibana:
+    image: docker.elastic.co/kibana/kibana-oss:7.1.1
+    container_name: doctrina_kibana
+    depends_on:
+      - elasticsearch
+    networks:
+      - doctrina_network
+    environment:
+      - SERVER_BASEPATH=/kibana
+
```

Comment:
upgrade to 8.4.2 both Kibana & elasticsearch. If any trouble, just target the version 7.17.6 for both as well

Reply:
Also, do you need Kibana in local ? If not, move the code to the infra